sqlmesh/core/model/meta.py

```diff
@@ -271,6 +289,7 @@ def _pre_root_validator(cls, values: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
     def _root_validator(cls, values: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
         values = cls._kind_validator(values)
         values = cls._normalize_name(values)
+        values = cls._normalize_default_catalog(values)
```

Comment:
I think this is wrong.

Please note that `_root_validator` is a `post` / `after` validator. At the same time you're using the `default_catalog` value in `pre` / `before` validators.

Reply:
Can you link me to an example where you think un-normalized `default_catalog` is being incorrectly used?

Reply:
I could've sworn I saw it in some other validator, but seems like you're actually using it before normalization just a few lines above: https://github.com/TobikoData/sqlmesh/blob/98b166f3cd98184c7b569fe0ec902c4ed179d68a/sqlmesh/core/model/meta.py#L261

Reply:
And that is correct right? Since if I used the normalized value then it would be a double normalization.